### ISSUE TITLE
missing label for SENSE dataset

### ISSUE THREAD

**Issue by linthmwan** [taohan10200/DRNet#8]
I download the SENSE dataset from baidu, but there is no label for scene IMG_4696_cut_03.txt at frame 000084.jpg and 000085.jpg. 
Can anyone provide GT of these two frames?
thanks!

**Comment by taohan10200** [comment]
- ...
- 000082.jpg 1280 720 511.866 4.628 527.381 18.839 521.197 9.768 13 
- 000083.jpg 1280 720 524.209 0 538.7 12.031 532.011 4.131 13 
- 000084.jpg 1280 720 
- 000085.jpg 1280 720 
- ...

Thanks for your reminder, it just because there is no object in these two frames.

**Comment by linthmwan** [comment]
[IMG_4696_cut_03.txt](https://github.com/taohan10200/DRNet/files/10709861/IMG_4696_cut_03.txt)
Above is the file I download, I also checked that there are people in the frames 84.jpg and 85.jpg exactly.

**Comment by taohan10200** [comment]
My file is the same with yours, I guess the annotators miss some objects in the whole dataset.

**Comment by linthmwan** [comment]
Thanks for reply!
I found this issue because of running the test_SENSE.py and it will have error at VIC.py line 206 
`match_gt, pois = self.get_ROI_and_MatchInfo(target[0], target[1],noise=None)`
and successively in points_from_den.py
`unmatched0 = torch.where(dis.min(1)[0]>0)[0]`
because there is no target.
Is there a correct way to run the test_SENSE.py?

**Comment by taohan10200** [comment]
The simple way is to skip this scene. We will try to fix it later.

**Comment by linthmwan** [comment]
Got it. Thank you for your patience.
